### DISCUSSION TITLE
How to merge a pull req

### DISCUSSION THREAD

**Discussion by jsMasterDND** [community/community#123791] [Repositories]
### Select Topic Area

Question

### Body

hey guys can somebody tell me how to megre a pull req in vs code

**Comment by IroshanBrian** [comment]
When you have selected the PR to close, you must click the Description section to open a new Tab in your VS Code editor.

In the center of the PR view, there’s a section related to closing the PR. Select which type of merge method you prefer and then click the “Merge Pull Request” button

**Comment by IroshanBrian** [comment]
Finally, edit or confirm the PR title and description and then click the button to confirm the merge.
